StitchCoreServicesTwilio/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,6 @@
+import XCTest
```

Comment:
Can remove this file?

Reply:
I think this file will be for when people run our tests on Linux?

Reply:
Beats me